build.gradle

```diff
@@ -745,7 +745,7 @@ ext.createDiffFile = { ->
   def file = Files.createTempFile(URLEncoder.encode(project.name, 'UTF-8'), '.diff').toFile()
   def diffBase = "${git.getUpstreamRemote()}/main"
   def command = [
-      'git', 'diff', diffBase, '--no-color', '--minimal', '--', '.'
+      'git', 'diff', "${diffBase}...", '--no-color', '--minimal', '--', '.'
```

Comment:
What's this change, as well as change in `git.gradle` for? Is that because of git version upgrade or something?

Reply:
It should make the coverage checks on our forks a bit more reliable. Currently, we take a diff between `upstream/main` and the branch being worked on. This means if `upstream/main` has PRs merged, then unless we rebase our branch, the diff will have many changes that are not touched by the code change in the branch.

The change in the patch above will only make the diff between the place where the branch branched off of till the head of the current working branch. So, it would mean only the changes that made by the current branch.

I think Github does some magic when running actions on PRs so that the diff is taken properly, but the same doesn't happen when we push the changes to our forks. So, it is possible that diff coverage generally passes in the PR, but fails on our forks

Reply:
This StackOverflow has more details: https://stackoverflow.com/a/40119935/1989520